components/NoUserRecipes.tsx

```diff
@@ -0,0 +1,35 @@
+import styled from 'styled-components';
+
+export default function NoUserRecipes() {
+  return (
+    <StyledArticle>
+      <StyledParagraph>
+        <StyledSpan>
+          Sorry :( <br />
+          You have not created any recipes yet.
+        </StyledSpan>
+      </StyledParagraph>
+      <StyledParagraph>You can create recipes here.</StyledParagraph>
```

Comment:
My first assumption when reading this was: "Oh, I can click here and directly create a recipe."

At least for me this a little bit misleading, but maybe it's just me.

Reply:
Good Point! We really forgot to add the link there :D We will do! Thanks!